pubspec.yaml

```diff
@@ -25,31 +25,31 @@ dependencies:
     sdk: flutter
 
   # UI
-  cupertino_icons: ^1.0.3
-  flutter_svg: ^0.22.0
+  cupertino_icons: ^1.0.4
+  flutter_svg: ^0.23.0+1
   nstack:
     git:
       url: git://github.com/nstack-io/flutter-sdk.git
       #ref: v0.2.5
 
   # Net
-  dio: ^4.0.0
+  dio: ^4.0.4
   jaguar_jwt: ^3.0.0
 
   # Utility
-  device_info_plus: ^2.1.0
-  package_info_plus: ^1.0.4
+  device_info_plus: ^3.1.1
+  package_info_plus: ^1.3.0
 
   # Persistence
-  shared_preferences: ^2.0.6
+  shared_preferences: ^2.0.9
 
   # Architecture
-  freezed: ^0.14.3
-  freezed_annotation: ^0.14.3
+  freezed: ^1.0.0
```

Comment:
Can we move `freezed` package to dev dependency? 

 [link](https://pub.dev/packages/freezed#install)

Reply:
👍